docs/tutorials/customize_models.md

```diff
@@ -170,14 +170,17 @@ custom_imports = dict(
 
 to the config file and avoid modifying the original code.
 
-#### 3. Modify the config file
+#### 3. Use the neck in your config file
```

Comment:
I find L59 also has typos, should be 'Use the voxel encoder in your config file'. Please also check others.